pkg/apic/subscription.go

```diff
@@ -36,6 +36,7 @@ type Subscription interface {
 	GetID() string
 	GetName() string
 	GetApicID() string
+	GetRemoteAPIAttributes() map[string]string
```

Comment:
We cannot rely on the `remoteAPIID` attribute any more for mulesoft, because that attribute reflects the API Service only, and not the revision. We are now able to generate traffic and create subscriptions for more than one revision of an API Service, and so we need to find the traffic and the subscriptions based on the api revision id, which we are publishing as the "API ID" attribute.